templates/common/copilot-gpt-basic/appPackage/declarativeCopilot.json.tpl

```diff
@@ -0,0 +1,4 @@
+{
+    "$schema": "https://graphdevxdata.blob.core.windows.net/data/schemas/CopilotGPTManifestSchema-1.0.json",
+    "actions": []
```

Comment:
I don't think we need actions now. This template is only for GPT basic now.

Reply:
I'm considering whether the `actions` should be left empty like this by default, meaning we won't include any action. This could let users know that they can add some actions using this field.

Reply:
Also, user can use a command to add an action based on our this template.

Reply:
Yeah, user can add action which we will generate for them. "actions should be left empty like this by default, meaning we won't include any action" - I think we should ask feedback from PM about this.

Reply:
Sure, let me check with PM